templates/zerver/api/changelog.md

```diff
@@ -20,6 +20,15 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 6.0
 
+
+**Feature level 131**
+
+* [`GET /events`](/api/get-events): Added `user_topic` event type
+  which is sent when a topic is muted or unmuted.
+* [`POST /register`](/api/register-queue): Added `user_topics` array
+  to the response.
+* [`GET /events`](/api/get-events), [`POST /register`](/api/register-queue): `muted_topics` array is no longer sent when user_topic is specified as an event type.
```

Comment:
> specified as an event type

Is this talking about `fetch_event_types`, or `event_types`, or both?

Reply:
@chrisbobbe  Yes, both. This should clearly state "when `user_topic` is specified in `fetch_event_types` or in `event_types` ", in all the places you've pointed out. Thanks.

Reply:
@chrisbobbe Yes, both.
For `POST /register`, `muted_topics` initial data array is not sent when `user_topic` is in `fetch_event_types` or `event_types` (as `event_types` is used to fetch initial state data if `fetch_event_types` is empty).
Similarly, for `GET /events`, the new `muted_topics` event (that contains the new `muted_topics` array) is not sent when `user_topic` is specified in `event_types`.
This should have been stated more clearly. I'll update the documentation accordingly for all the places you've pointed out above. Thanks!